.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+  build_and_deploy:
+    env:
+        BASE_VERSION: 5.0.0.2012
```

Comment:
let's add a comment here saying "latest non-preview version at the time of writing this CI script"

Reply:
> Last thing: before I merge this PR you need to give me access to https://www.nuget.org/packages/DotNetForms.Platform.GTK so I can create a nuget api key.

I gave you access 2-3 days ago I think